docs/cluster_install.md

```diff
@@ -374,6 +399,30 @@ This content describes how to install the Open Horizon agent on k3s or microk8s
     ./agent-install.sh -D cluster -i anax: -c css: -k css: --namespace $AGENT_NAMESPACE --namespace-scoped
     ```
 
+    **Note** If your server is not configured with the certificate, config, and agent files you will need to use the following alternative steps to install:
+
+    a. Create an empty certificate:
+
+    ```bash
+    touch agent-install.crt
+    ```
+
+    b. Create a local install configuration file named `agent-install.cfg` containing (change "http" to "https" if secure transport is configured):
+
+    ```text
+    HZN_EXCHANGE_URL=http://<ip-address-here>:3090/v1
+    HZN_FSS_CSSURL=http://<ip-address-here>:9443/
+    HZN_AGBOT_URL=http://<ip-address-here>:3111
+    HZN_SDO_SVC_URL=<ip-address-here>:9008/api
+    HZN_FDO_SVC_URL=<ip-address-here>:9008/api
+    ```
+
+    c. Install a cluster-scoped agent using the local configuration:
+
+    ```bash
+    ./agent-install.sh -D cluster -i anax: -c css: -k ./agent-install.cfg --namespace $AGENT_NAMESPACE
+    ```
+
 7. Verify that the agent pod is running:
 
     ```bash
```

Comment:
not part of your PR but the examples above show setting the AGENT_NAMESPACE env variable.... but here we tell them if not set to use `openhorizon-agent`...  I think we should stick with the $AGENT_NAMESPACE syntax to be consistent throughout